### PR TITLE
Fix typos in sample folder

### PR DESCRIPTION
**Pull request by davydovanton**

### CODE REVIEW

File: sample/trick2013/kinaba/remarks.markdown [hunk 1]
```diff
@@ -20,7 +20,7 @@ The algorthim is the obvious loop "32.upto(126){|x| putc x}".
 
 It is not so hard to transform it to use each character *at most once*. Only the slight difficulty comes from the constraint that we cannot "declare and then use" variables, because then the code will contain the variable name twice. This restriction is worked around by the $. global variable, the best friend of Ruby golfers.
```
Comment (bryndyment) [inline sample/trick2013/kinaba/remarks.markdown]: While you're at it, should be `The only`, not `Only the` ...

File: sample/trick2013/kinaba/remarks.markdown [hunk 2]
```diff
@@ -20,7 +20,7 @@ The algorthim is the obvious loop "32.upto(126){|x| putc x}".
...
 
-The relatively interesting part is to use all the charcters *at least once*. Of course, this is easily accomplished by putting everything into a comment (i.e., #unsed...) or to a string literal (%(unused...), note that normal string literals are forbidden since they use quotation marks twice). Hey, but that's not fun at all! I tried to minimize the escapeway.
+The relatively interesting part is to use all the charcters *at least once*. Of course, this is easily accomplished by putting everything into a comment (i.e., #unused...) or to a string literal (%(unused...), note that normal string literals are forbidden since they use quotation marks twice). Hey, but that's not fun at all! I tried to minimize the escapeway.
 
 * "@THEqQUICKbBROWNfFXjJMPSvVLAZYDGgkyz". Trash box of unused alphabet. I with I could have used "gkyz" somewhere else.
```
Comment (bryndyment) [inline sample/trick2013/kinaba/remarks.markdown]: ... and `wish`, not `with`.

**Comment by davydovanton** [comment]
@bryndyment nice catch, thanks!

**Comment by hsbt** [comment]
Fixed at https://github.com/ruby/ruby/commit/a45972f2a89d4b8c2b6c346a547140e8c3a4218d and https://github.com/ruby/ruby/commit/d6b8acacf60ac497815f3c8f33e10a189b00a34a

**Comment by davydovanton** [comment]
@hsbt thanks! :star2:

**Pull request closed without merging**